### ISSUE TITLE
[BUG] Docstring causing `invalid bcc code`

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1063]
It seems like having a docstring in a class method causes `invalid bcc code` to be thrown while obfuscating.

**Executed command:** `pyarmor -d gen --enable-bcc .\pyarmor_test\bcc_test.py`

**bcc_test.py**
```python
class BCCError:
    def test(self):
        """
        This is a test
        """
        pass

```

**pyarmor.error.log**
```
12296 MainProcess 2023-03-31 23:01:08,893
Traceback (most recent call last):
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 629, in main
    main_entry(sys.argv[1:])
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 617, in main_entry
    args.func(ctx, args)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 179, in cmd_gen
    builder.process(options, pack=args.pack)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\generate.py", line 153, in process
    self._obfuscate_scripts()
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\generate.py", line 118, in _obfuscate_scripts
    code = Pytransform3.generate_obfuscated_script(self.ctx, r)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\core\__init__.py", line 48, in generate_obfuscated_script
    return m.generate_obfuscated_script(ctx, res)
  File "<maker>", line 719, in generate_obfuscated_script
  File "<maker>", line 522, in process
  File "<maker>", line 522, in <listcomp>
  File "<maker>", line 1255, in handle
  File "<maker>", line 1253, in travel
  File "<maker>", line 1253, in travel
  File "<maker>", line 1250, in travel
  File "<maker>", line 1222, in _patch_co_object
  File "<maker>", line 1199, in _patch_co_code
RuntimeError: invalid bcc code
```

Removing the `This is a test` docstring fixes the problem.

**Comment by jondy** [comment]
Thank you very much, reproduced and fixed in next release.

**Comment by jondy** [comment]
Fixed in dev version `8.1.dev1`

**Comment by AlecHaring** [comment]
Confirmed, fixed
